connect-examples/v2/csharp_payment/Startup.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace sqRazorSample
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc()
+                    .AddRazorPagesOptions(o =>
+                    {
+                        // This is for Sample only, remove this to handle the post request in secure way
+                        o.Conventions.ConfigureFilter(new IgnoreAntiforgeryTokenAttribute());
+                    });
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+            else
+            {
+                app.UseExceptionHandler("/Error");
+            }
+
+            app.UseStaticFiles();
+
+            app.UseMvc();
+
+            Square.Connect.Client.Configuration.Default.AccessToken = this.Configuration["AppSettings:AccessToken"];

```

Comment:
Should we add a comment here to explain why we are doing this?

Reply:
Sure! Will this suffice:
// This is globally setting the Square's API access token. 
// This access token will be used for all calls within the application.